### PR TITLE
Wfnews 1074 update notifications-db terraform

### PR DESCRIPTION
**Pull request by yzlucas**

### CODE REVIEW

**Comment by yzlucas** [comment]
we may need "NOTIFICATIONS_LIQUIBASE_IMAGE " variable in the github action. 
However I am not sure if it would be the same as  "LIQUIBASE_IMAGE"? do you think it is necessary? @vivid-cpreston

File: terraform/ecs.tf [hunk 1]
```diff
@@ -1357,4 +1357,146 @@ resource "aws_ecs_service" "apisix_gui" {
...
+      memory      = var.server_memory
+      networkMode = "awsvpc"
+      portMappings = [
+        {
+          protocol      = "tcp"
+          containerPort = var.db_port
+          hostPort      = var.db_port
+        }
+      ]
+      environment = [
```
Comment (vivid-cpreston) [inline terraform/ecs.tf:1407]: Need to include a value for CHANGELOG_FOLDER here. Also need to update wfnews_liquibase resource to include a value for CHANGELOG_FOLDER

**Review by vivid-cpreston — CHANGES_REQUESTED** [review]

File: terraform/ecs.tf [hunk 2]
```diff
@@ -1357,4 +1361,152 @@ resource "aws_ecs_service" "apisix_gui" {
...
+      volumesFrom = []
+    }
+  ])
+}
+
+resource "aws_ecs_service" "notifications_liquibase" {
+  count                             = 1
+  name                              = "notifications-liquibase-service-${var.target_env}"
+  cluster                           = aws_ecs_cluster.wfnews_main.id
+  task_definition                   = aws_ecs_task_definition.wfnews_tasks[wfnews_liquibase].arn
```
Comment (vivid-cpreston) [inline terraform/ecs.tf:1481]: This should be aws_ecs_task_definition.notifications_liquibase.arn - wfnews_tasks[wfnews_liquibase].arn would be for if the task properties had been added to the tasks map near the top of this file

File: terraform/ecs.tf [hunk 3]
```diff
@@ -1357,4 +1361,152 @@ resource "aws_ecs_service" "apisix_gui" {
...
+      portMappings = [
+        {
+          protocol      = "tcp"
+          containerPort = var.db_port
+          hostPort      = var.db_port
+        }
+      ]
+      environment = [
+        {
+        {
```
Comment (vivid-cpreston) [inline terraform/ecs.tf:1413]: syntax errror here - you have an extra layer of {}

File: terraform/variables.tf [hunk 4]
```diff
@@ -351,12 +357,24 @@ variable "liquibase_container_name" {
...
+  type        = string
+}
+
 variable "liquibase_image" {
   description = "Full name of liquibase image"
   type        = string
   default     = ""
 }
 
+variable "notificatinos_liquibase_image" {
```
Comment (vivid-cpreston) [inline terraform/variables.tf:372]: I don't think this is needed any more

File: terraform/ecs.tf [hunk 5]
```diff
@@ -1357,4 +1361,152 @@ resource "aws_ecs_service" "apisix_gui" {
 
   tags = local.common_tags
 }
+
+resource "aws_ecs_task_definition" "notifications_liquibase" {
```
Comment (vivid-cpreston) [inline terraform/ecs.tf:1365]: This is currently in the commented-out section - is this intended?

**Review by vivid-cpreston — CHANGES_REQUESTED** [review]
A few more changes I noticed

**Review by vivid-cpreston — APPROVED** [review]

**Pull request merged**